pyslackersweb/__init__.py

```diff
@@ -15,27 +20,43 @@
 
 
 sentry_sdk.init(
-    dsn=os.getenv("SENTRY_DSN"),
+    dsn=settings.SENTRY_DSN,
     integrations=[
         AioHttpIntegration(),
         LoggingIntegration(level=logging.INFO, event_level=logging.ERROR),
     ],
-    release=os.getenv("PLATFORM_TREE_ID"),
-    environment=os.getenv("PLATFORM_BRANCH"),
+    release=settings.SENTRY_RELEASE,
+    environment=settings.SENTRY_ENVIRONMENT,
 )
 
 
-async def index(request: web.Request) -> web.Response:
+async def index(request: web.Request) -> web.HTTPFound:
     location = request.app["website_app"].router["index"].url_for()
     return web.HTTPFound(location=location)
 
 
 async def app_factory() -> web.Application:
-    app = web.Application()
+    app = web.Application(
+        middlewares=[
+            ForwardedRelaxed().middleware,
+            XForwardedRelaxed().middleware,
+            request_context_middleware,
+        ]
+    )
+    app.update(  # pylint: disable=no-member
+        client_session=ClientSession(),
+        scheduler=AsyncIOScheduler(),
+        redis=await aioredis.create_redis_pool(settings.REDIS_URL),
+    )
+
+    app.cleanup_ctx.append(apscheduler)
+    app.on_cleanup.extend([close_client_session, close_redis])
+
     app.router.add_get("/", index)
 
-    website_app = website.app_factory()
+    app["website_app"] = website_app = await website.app_factory(
```

Comment:
I'm not  a big fan of that syntax tbh. Maybe just get rid of the `website_app` ?